service/src/main/resources/application.yml

```diff
@@ -69,7 +71,8 @@ wds:
 
 leonardo:
   baseUri: "https://leonardo.dsde-dev.broadinstitute.org/"
-  wdsAppTypeNames: ['WDS', 'CROMWELL']
+  wdsAppTypeNames: ['WDS', 'CROMWELL'] # keeping CROMWELL type for backwards compatibility
```

Comment:
I think it might be helpful to add a comment on what these values are for since the apps are listed in priority order for finding proxy url